g2p/mappings/langs/font-encodings/myh_doulos.json

```diff
@@ -0,0 +1,282 @@
+[
+  {
+    "in": "\\ ",
+    "out": " "
+  },
+  {
+    "in": "\"",
+    "out": "’"
+  },
+  {
+    "in": "\\#",
+    "out": "‘"
+  },
+  {
+    "in": "\\)",
+    "out": "*"
+  },
+  {
+    "in": "\\*",
+    "out": "+"
+  },
+  {
+    "in": "\\+",
+    "out": ","
+  },
+  {
+    "in": ",",
+    "out": "-"
+  },
+  {
+    "in": "\\-",
+    "out": "."
+  },
+  {
+    "in": "\\.",
+    "out": "/"
+  },
+  {
+    "in": "4",
+    "out": "ă·"
+  },
+  {
+    "in": "5",
+    "out": "ĭ·"
+  },
+  {
+    "in": "6",
+    "out": "ŭ·"
+  },
+  {
+    "in": "9",
+    "out": ":"
+  },
+  {
+    "in": ";",
+    "out": "<"
+  },
+  {
+    "in": "<",
+    "out": "="
+  },
+  {
+    "in": "=",
+    "out": ">"
+  },
+  {
+    "in": ">",
+    "out": "?"
+  },
+  {
+    "in": "\\?",
+    "out": "°"
+  },
+  {
+    "in": "@",
+    "out": "a·"
+  },
+  {
+    "in": "B",
+    "out": "c̓"
+  },
+  {
+    "in": "C",
+    "out": "č"
+  },
+  {
+    "in": "D",
+    "out": "e·"
+  },
+  {
+    "in": "E",
+    "out": "̓č̓"
+  },
+  {
+    "in": "F",
+    "out": "Ɂ"
+  },
+  {
+    "in": "G",
+    "out": "ḥ"
+  },
+  {
+    "in": "H",
+    "out": "i·"
+  },
+  {
+    "in": "I",
+    "out": "ƛ̓"
+  },
+  {
+    "in": "J",
+    "out": "k̓"
+  },
+  {
+    "in": "K",
+    "out": "ɫ"
+  },
+  {
+    "in": "L",
+    "out": "m̓"
+  },
+  {
+    "in": "M",
+    "out": "n̓"
+  },
+  {
+    "in": "N",
+    "out": "o·"
+  },
+  {
+    "in": "O",
+    "out": "p̓"
+  },
+  {
+    "in": "P",
+    "out": "q̓"
+  },
+  {
+    "in": "Q",
+    "out": "ƛ"
+  },
+  {
+    "in": "R",
+    "out": "š"
+  },
+  {
+    "in": "S",
+    "out": "t̓"
+  },
+  {
+    "in": "T",
+    "out": "u·"
+  },
+  {
+    "in": "U",
+    "out": "T"
+  },
+  {
+    "in": "V",
+    "out": "w̓"
+  },
+  {
+    "in": "W",
+    "out": "x̌"
+  },
+  {
+    "in": "X",
+    "out": "y̓"
+  },
+  {
+    "in": "Y",
+    "out": "̴Ɂ"
+  },
+  {
+    "in": "\\]",
+    "out": "ʷ"
+  },
+  {
+    "in": "\\^",
+    "out": "_"
+  },
+  {
+    "in": "`",
+    "out": "a"
+  },
+  {
+    "in": "a",
+    "out": "b"
+  },
+  {
+    "in": "b",
+    "out": "c"
+  },
+  {
+    "in": "c",
+    "out": "d"
+  },
+  {
+    "in": "d",
+    "out": "e"
+  },
+  {
+    "in": "f",
+    "out": "g"
+  },
+  {
+    "in": "g",
+    "out": "h"
+  },
+  {
+    "in": "h",
+    "out": "i"
+  },
+  {
+    "in": "i",
+    "out": "ʷ"
+  },
+  {
+    "in": "j",
+    "out": "k"
+  },
+  {
+    "in": "k",
+    "out": "l"
+  },
+  {
+    "in": "l",
+    "out": "m"
+  },
+  {
+    "in": "m",
+    "out": "n"
+  },
+  {
+    "in": "n",
+    "out": "o"
+  },
+  {
+    "in": "o",
+    "out": "p"
+  },
+  {
+    "in": "p",
+    "out": "q"
+  },
+  {
+    "in": "q",
+    "out": "r"
+  },
+  {
+    "in": "r",
+    "out": "s"
+  },
+  {
+    "in": "s",
+    "out": "t"
+  },
+  {
+    "in": "t",
+    "out": "u"
+  },
+  {
+    "in": "u",
+    "out": "P"
+  },
+  {
+    "in": "v",
```

Comment:
Why are there ASCII characters mapped to other ASCII characters as part of this mapping? Surely that's not part of the font-encoding?

Reply:
I see prevent-feeding is true globally. Is it really the case that this font renders `s` as `t` and `t` as `u`?? Why!?

Reply:
Amazingly enough, yes - try copying some text from https://depts.washington.edu/wll2/files/davidson_02_diss.pdf - no idea if something went weird in that document in particular, if you have some other Nuuchahnulth documents with SIL fonts in them it would be good to check.

Reply:
The PDF mapping to "Unicode" characters is very weird, no idea at all how it got that way:
![Screenshot from 2024-04-02 07-46-50](https://github.com/roedoejet/g2p/assets/3325008/9400276f-e259-4d03-8778-211af8b0affe)